contrib/hyperscan/matching/input_matchers/test/matcher_speed_test.cc

```diff
@@ -43,11 +43,9 @@ BENCHMARK(BM_CompiledGoogleReMatcher);
 
 // NOLINTNEXTLINE(readability-identifier-naming)
 static void BM_HyperscanMatcher(benchmark::State& state) {
-  envoy::extensions::matching::input_matchers::hyperscan::v3alpha::Hyperscan config;
-  auto regex = config.add_regexes();
-  regex->set_regex(std::string(cluster_re_pattern));
   auto instance = ThreadLocal::InstanceImpl();
-  auto matcher = Extensions::Matching::InputMatchers::Hyperscan::Matcher(config, instance);
+  auto matcher = Extensions::Matching::InputMatchers::Hyperscan::Matcher(
+      {cluster_re_pattern.data()}, {0}, {0}, instance, false);
```

Comment:
pre-existing, but you can s/cluster_re_pattern/ClusterRePattern/ per Envoy naming convention for constants?

More importantly, if a null-terminated char* is expected here you can't guarantee you'll get that with .data(). Strictly speaking you need to make use `std::string(ClusterRePattern).c_str()` here.

Reply:
Thanks, fixed.